src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Cmdlets/UpdateAzureRmManagementGroup.cs

```diff
@@ -0,0 +1,87 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ManagementGroups.Common;
+using Microsoft.Azure.Commands.ManagementGroups.Models;
+using Microsoft.Azure.Management.ManagementGroups;
+using Microsoft.Azure.Management.ManagementGroups.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Cmdlets
+{
+    [Cmdlet("Update", "AzureRmManagementGroup", DefaultParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, SupportsShouldProcess = true), OutputType(typeof(PSManagementGroupNoChildren))]
+    public class UpdateAzureRmManagementGroup : AzureManagementGroupsCmdletBase
+    {
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.InputObject, ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public PSManagementGroup ManagementGroup { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupNoChildrenParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.InputObject, ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public PSManagementGroupNoChildren ManagementGroupNoChildren { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = true, HelpMessage = Constants.HelpMessages.GroupName, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        public string GroupName { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.DisplayName)]
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupNoChildrenParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.DisplayName)]
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.DisplayName)]
+        [ValidateNotNullOrEmpty]
+        public string DisplayName { get; set; } = null;
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.ParentId)]
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupNoChildrenParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.ParentId)]
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet, Mandatory = false,
+            HelpMessage = Constants.HelpMessages.ParentId)]
+        [ValidateNotNullOrEmpty]
+        public string ParentId { get; set; } = null;
```

Comment:
@rajshah11 same question about whether or not it makes sense to allow the user to pass the `PSManagementGroup` object they want to be the parent

Reply:
Will make the change.